nshlib/Kconfig

```diff
@@ -857,13 +832,75 @@ config NSH_FATMOUNTPT
 		will mount a FAT FS under /tmp. This is the location where the FAT
 		FS will be mounted.  Default is "/tmp".
 
+endif # NSH_ROMFSETC
+
+config NSH_RUNSYSINITSCRIPT
```

Comment:
the new option will break all config

Reply:
Yes it will. Do you have any idea to avoid that?

Reply:
I'm cancelling this PR. I will come up with a new way to do it.